src/main/java/com/microsoft/azure/springcloudplayground/generator/ProjectGenerator.java

```diff
@@ -45,6 +47,19 @@
     @Setter
     private transient Map<String, List<File>> temporaryFiles = new LinkedHashMap<>();
 
+    private static final Map<String, List<String>> MODULE_TO_TEMPLATE;
```

Comment:
What's the problem with original dynamic loading all templates under certain directory? I think the former way is more flexible

Reply:
We can write another `write` directory for this.

Reply:
Agree.